package-lock.json

```diff
@@ -11,9 +11,19 @@
       "dependencies": {
         "@nestjs/common": "^10.0.0",
```

Comment:
Package.lock.json should be added to a git ignore file, we dont need that on github.
Because it holds the local libraries on your machine it does not mean all of the team should have the same versioning as u